build.gradle

```diff
@@ -38,7 +38,7 @@ group = "com.github.eilslabs"
 
```

Comment:
Can't comment line 31 directly. Please get rid of eilslabs related entries if possible.

Reply:
fixed

Reply:
There are lots of copyright notices etc. containing eilslabs. We can change these in one go later.